### PR TITLE
Show bbgithub error, bug fix

### PR DESCRIPTION
**Pull request by markroseberry**
*Issue number of the reported bug or feature request: #RDISCROWD-2528*

**Describe your changes**
Bug fix for displaying last task presenter push details.
Add support for displaying an error message in the push details.

**Testing performed**
Manual

### CODE REVIEW

File: templates/projects/task_presenter_editor.html [hunk 1]
```diff
@@ -143,6 +143,12 @@
                                 <td><p>Last Commit:</p></td>
                                 <td class="ref-details"><a href={{ guidelines_github.ref_url }} target="_blank">{{ guidelines_github.ref }}</a></td>
                             </tr>
+                            {% if guidelines_github.error %}
```
Comment (dchhabda) [inline templates/projects/task_presenter_editor.html]: As guidelines_github_exists can be set to true based on multiple factors that may not be set when there's error, can this be outside of {% if guidelines_github_exists %} block and be instead
> {% if guidelines_github and guidelines_github.error %}

Same for task_presenter_github_exists

Comment (markroseberry) [reply templates/projects/task_presenter_editor.html]: How about this so I can fit it all in one table?

Comment (dchhabda) [reply templates/projects/task_presenter_editor.html]: technically yes, however meaning of guidelines_github_exists changes to guidelines_github_exists_or_error

Comment (markroseberry) [reply templates/projects/task_presenter_editor.html]: Now it's perfect.

**Pull request merged**